src/views/createCommunity/index.tsx

```diff
@@ -531,6 +535,7 @@ function CreateCommunityScreen() {
                     }
                     setSending(false);
                     setSendingSuccess(true);
+                    navigation.navigate(Screens.CommunityManager);
```

Comment:
Could we, instead of navigating to a new page, drop the current one? Like, go back and then navigate? I prefer that it won't stay on the navigation history.